src/construction.jl

```diff
@@ -29,15 +35,16 @@ function load_data(config::Dict, starttime::DateTime, endtime::DateTime)
     profile = filter(:id => in(ids), profile; view = true)
 
     # for forcing first get the right time range out
+    @assert issorted(forcing.time)
     startrow = searchsortedfirst(forcing.time, starttime)
     endrow = searchsortedlast(forcing.time, endtime)
     forcing = @view forcing[startrow:endrow, :]
     # then keep only IDs we use
     forcing = filter(:id => in(ids), forcing; view = true)
 
+    # TODO Is order required for ids?
     @assert issorted(profile.id)
     @assert issorted(static.id)
```

Comment:
For the config setting `ids`, it does not need to be ordered. In the static and profile tables, they do need to be sorted currently, otherwise these two lines will return incorrect results:

```julia
profile_rows = searchsorted(df.id, id)
rows = searchsorted(static.id, id)
```

For profile it is documented, but I see in https://deltares.github.io/Ribasim.jl/#static it is not mentioned, do you mind adding that as well to the docs?